deployment-templates/compute-engine/forseti-instance.py.schema

```diff
@@ -80,3 +80,15 @@ properties:
     type: integer
     default: 3306
     description: The port for the database.
+
+  network-host-project:
+    type: string
+    description: the id of the shared network host project
+
+  vpc-name:
+    type: string
+    description: the name of the shared network
+
+  subnetwork:
+    type: string
+    description: the name of the subnet
```

Comment:
ditto: Please capitalize ```The``` at the start of the description, and add a ```.``` at the end of the sentence.

Reply:
Complete